`make` command require `swagger` cli

Trying to run `make` produce an error due to `swagger` not being found.

```
$ make
swagger generate server -f openapi.yaml -q -r COPYRIGHT.txt -t pkg/generated --exclude-main -A fulcio_server --exclude-spec --flag-strategy=pflag -P github.com/coreos/go-oidc/v3/oidc.IDToken
make: swagger: Command not found
Makefile:10: recipe for target 'pkg/generated/models/%.go' failed
make: *** [pkg/generated/models/%.go] Error 127
```

I'm not sure how I'm supposed to install it, I'd like some guidance. Happy to improve the docs with a PR. 
Thanks!

--- a/README.md
+++ b/README.md
@@ -29,6 +29,17 @@ The fulcio root cert is currently:
 
 We **WILL** change this and add intermediaries in the future.
 
+## Build for development
+
+After cloning the repository:
+
+```
+$ go get -u github.com/go-swagger/go-swagger/cmd/swagger@v0.26.1
+$ make
+```
+
+There are other targets available in the [`Makefile`](Makefile), check it out.
+
 ## API
 
 The API is defined via OpenAPI, defined [here](openapi.yaml).

